app/kafka/manager/KafkaManager.scala

```diff
@@ -232,13 +232,38 @@ class KafkaManager(akkaConfig: Config) {
           Future.successful[ApiError \/ TopicIdentity](-\/(ApiError(s"Topic not found $topic for cluster $clusterName")))
         } { td =>
           tryWithKafkaManagerActor(KMClusterQueryRequest(clusterName, KSGetBrokers)) { brokerList: BrokerList =>
-            TopicIdentity.from(brokerList, td)
+            val metrics = getTopicMetrics(clusterName, topic)
+            TopicIdentity.from(brokerList, td, Some(metrics))
           }
         }
       })
     }
   }
 
+  def getTopicMetrics(clusterName: String, topic: String) : TopicMetrics = {
```

Comment:
We should create an actor which periodically grabs JMX stats and keeps them in-memory.  The UI would just use KafkaManager to grab the current in-memory state of the metrics.